lib/fennec/stun.ex

```diff
@@ -17,15 +18,23 @@ defmodule Fennec.STUN do
     {:ok, {binary, %TURN{}}} | {:ok, :void} | {:error, reason :: term}
```

Comment:
I think we need to update this spec. This function can also return `{:ok, {:void, %TURN{}}}`. Also I don't think it returns `{:ok, :void}` and event if did then we should change that because it's still better to just return `{:ok, {:void, %TURN{}}}` in this case.